faster_fifo.pyx

```diff
@@ -154,6 +151,21 @@ class Queue:
         return messages
 
     def reallocate_msg_buffer(self, new_size):
-        log.info('Reallocating msg buffer size: %d', new_size)
+        # log.info('Reallocating msg buffer size: %d', new_size)
         self.message_buffer = (ctypes.c_ubyte * new_size)()
         self.message_buffer_memview = memoryview(self.message_buffer)
+
+    def qsize(self):
+        return Q.get_queue_size(<void *>q_addr(self))
+
+    def empty(self):
+        return Q.get_queue_size(<void *>q_addr(self)) == 0
+    
+    def full(self):
+        return Q.get_queue_size(<void *>q_addr(self)) == self.max_size_bytes
+
+    def join_thread(self):
```

Comment:
These functions should not return anything.
Look at the implementation of join_thread of multiprocessing.Queue. It has no return value, so users won't expect these methods to return strings. Please just replace this return with a docstring, and then use `pass` to indicate an empty function like:

```
def join_thread(self):
    """This is not implemented as this implementation does not use a background thread"""
    pass
```